express-get-json/index.js

```diff
@@ -0,0 +1,34 @@
+const express = require('express');
+
+const grades = {
+  12: {
+    id: 12,
+    name: 'Tim Berners-Lee',
+    course: 'HTML',
+    score: 95
+  },
+  47: {
+    id: 47,
+    name: 'Brendan Eich',
+    course: 'JavaScript',
+    score: 100
+  },
+  273: {
+    id: 273,
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92
+  }
+};
+
+const app = express();
+
+app.get('/api/grades', (req, res) => {
+
+  res.json([grades]);
+
+});
```

Comment:
Your server's `GET /api/grades` response should be an array of objects, not an array of an object of objects:

---

![target reference](https://user-images.githubusercontent.com/16143771/176979658-01802e9a-0edc-41c7-94aa-a7f2c6a85b60.png)

---

![](https://user-images.githubusercontent.com/91721272/176979494-b186063c-e320-4dbb-a668-a94eb1e686a3.png)